devel/scanprov

```diff
@@ -166,6 +169,12 @@ if ($write) {
 
     # Only a few files will have exceptions that apply to them.  Rewrite each
     foreach my $version (keys %add_by_version) {
+        if (is_devel_release($version)) {
+            my ($super, $major, $minor) = parse_version($version);
+            $major++;   # Go to next highest version that isn't a deve
+            $version = "$super.$major.0";
```

Comment:
maybe also ensure we only check that version once?

Reply:
I thought you had caught something here, but in looking further, I don't think so.  $version is changed only once per iteration.  Then I thought maybe you meant that the hash gets modified by incrementing $version, but an experiment shows that $version is a copy of the key.

This is a subtlety I had never considered before:  foreach my $element (@array) delivers a pointer to the actual element so that modifying it changes @array, but keys %hash delivers a copy of the key.

Reply:
thanks for your reply this is fine